tests/perf/run_perf_tests.py

```diff
@@ -7,69 +7,80 @@
-            self.vegeta_pattern_tar_file = argv[1]
-            self.daemon_on_core = argv[2]
-            self.tg_addr = argv[3]
-            self.geth_homedir = argv[4]
-        elif len(argv) == 6:
-            self.vegeta_pattern_tar_file = argv[1]
-            self.daemon_on_core = argv[2]
-            self.tg_addr = argv[3]
-            self.geth_homedir = argv[4]
-            self.repetitions = int(argv[5])
-        elif len(argv) == 7:
-            self.vegeta_pattern_tar_file = argv[1]
-            self.daemon_on_core = argv[2]
-            self.tg_addr = argv[3]
-            self.geth_homedir = argv[4]
-            self.repetitions = int(argv[5])
-            self.test_sequence = argv[6]
-        else:
-            print("Usage: " + argv[0] + " vegetaPatternTarFile [daemonOnCore] [turboGethAddress] [turboGethHomeDir] [testRepetitions] [testSequence]")
-            print("")
-            print("Launch an automated performance test sequence on Silkrpc and RPCDaemon using Vegeta")
-            print("")
-            print("vegetaPatternTarFile     path to the request file for Vegeta attack")
-            print("daemonOnCore             logical cpu list in taskset format (e.g. - or 0-1 or 0-2 or 0,2...)              [default: " + DEFAULT_DAEMON_ON_CORE +"]")
-            print("turboGethAddress         address of TG Core component as <address>:<port> (e.g. localhost:9090)           [default: " + DEFAULT_TG_ADDRESS + "]")
-            print("turboGethHomeDir         path to TG home folder (e.g. ../../../turbo-geth/)                               [default: " + DEFAULT_GETH_HOME_DIR + "]")
-            print("testRepetitions          number of repetitions for each element in test sequence (e.g. 10)                [default: " + str(DEFAULT_REPETITIONS) + "]")
-            print("testSequence             list of query-per-sec and duration tests as <qps1>:<t1>,... (e.g. 200:30,400:10) [default: " + DEFAULT_TEST_SEQUENCE + "]")
+        try:
+            opts, _ = getopt.getopt(argv[1:], "vp:c:a:g:s:r:t:")
+        except getopt.GetoptError as err:
+            # print help information and exit:
+            print(err)
+            usage(argv)
             sys.exit(-1)
 
+        for option, optarg in opts:
+            if option in ("-h", "--help"):
+                usage(argv)
+                sys.exit(-1)
+            elif option == "-p":
+                self.vegeta_pattern_tar_file = optarg
+            elif option == "-c":
+                self.daemon_vegeta_on_core = optarg
+            elif option == "-a":
+                self.tg_addr = optarg
+            elif option == "-g":
+                self.geth_builddir = optarg
+            elif option == "-s":
+                self.silkrpc_build_dir = optarg
+            elif option == "-r":
+                self.repetitions = int(optarg)
+            elif option == "-t":
+                self.test_sequence = optarg
+            else:
+                usage(argv)
+                sys.exit(-1)
```

Comment:
Move lines 63-83 within `try` block after line 57 for better program flow separation